OutOfSchool/OutOfSchool.WebApi/Models/ChildDTO.cs

```diff
@@ -10,17 +10,17 @@ public class ChildDto
         public long Id { get; set; }
 
         [Required(ErrorMessage = "First name is required")]
-        [MaxLength(40)]
-        [RegularExpression(@"[\w\-\']*", ErrorMessage = "First name cannot contains digits")]
+        [StringLength(30, MinimumLength = 1)]
+        [RegularExpression(@"^[А-ЯҐЄІЇ](([\'\-][А-ЯҐЄІЇа-яґєії])?[а-яґєії]*)*$", ErrorMessage = "First name contains invalid characters")]
```

Comment:
Do we really need to enforce first capital letter?
p.s. question for discussion, not immediate action.